client/client.go

```diff
@@ -43,11 +44,11 @@ const (
 var (
 	// ErrClientNotActivated occurs when an inactive client executes a function
 	// that can only be executed when activated.
-	ErrClientNotActivated = errors.New("client is not activated")
+	ErrClientNotActivated = goerrors.New("client is not activated")
 
 	// ErrDocumentNotAttached occurs when the given document is not attached to
 	// this client.
-	ErrDocumentNotAttached = errors.New("document is not attached")
+	ErrDocumentNotAttached = goerrors.New("document is not attached")
```

Comment:
👋  I came here from https://github.com/pkg/errors/issues/242

I tend to use `fmt.Errorf` for declaring error variables so the only `errors` package in the codebase is `github.com/pkg/errors`. This way I don't have to think about which `errors` package I'm using, it is never std errors.